### PR TITLE
Payments section design

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

Payments section
- Add general payment info and payment FAQ link
- Show monthly total, old gross if there's a discount and a free period if any
- Make Trustly UI disappear when connection is complete

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by alebedev** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3356** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3356" target="_blank">Graphite</a>  👈
    * **PR #3357** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3357" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3356?utm_source=stack-comment).

File: apps/store/src/features/memberArea/PaymentsSection/PaymentsSection.tsx [hunk 1]
```diff
@@ -1,39 +1,85 @@
...
+    <>
       <Heading as="h3" variant="standard.24">
         Payment connection
       </Heading>
       {currentMember.hasActivePaymentConnection ? (
         <>
-          <Text>✅&nbsp;Configured</Text>
-          <PaymentConfiguration startButtonText={'Change'} onSuccess={refetch} />
+          <Text>✅&nbsp;Connected</Text>
+          <PaymentConfiguration startButtonText="Change connection" />
```
Comment (gustaveen) [inline apps/store/src/features/memberArea/PaymentsSection/PaymentsSection.tsx:74]: nit:
```suggestion
          <PaymentConfiguration startButtonText="Change payment connection" />
```

**Review by gustaveen — APPROVED** [review]
Great!

**Comment by alebedev** [comment]
### Merge Activity

* **Oct 6, 5:20 AM**: @alebedev merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3356).

**Pull request merged**